handbook/benefits.md

```diff
@@ -312,17 +380,17 @@ Employees may be asked to provide a Fitness for Duty form completed by their med
 
 Employees who need to take dedicated time off to care for a seriously ill or injured person will be allowed to take up to 4 weeks of paid Caregiver Leave. The illness or injury must be a serious health condition (defined below).
```

Comment:
For above line 305/373 - Vesting. Remove "Vesting will be paused during unpaid leave"